articles/azure-monitor/app/opencensus-python.md

```diff
@@ -56,7 +56,11 @@ First you have to create an Application Insights resource, which will generate a
     > [!NOTE]
     > `python -m pip install opencensus-ext-azure` assumes that you have a PATH environment variable set for your Python installation. If you have not configured this, you would need to give the full directory path to where your Python executable is located which would result in a command like: `C:\Users\Administrator\AppData\Local\Programs\Python\Python37-32\python.exe -m pip install opencensus-ext-azure`.
 
-2. First let's generate some trace data locally. In Python IDLE, or your editor of choice, enter the following code:
+2. The SDK utilizes three Azure Monitor exporters to send different types of telemetry to application insights: trace, metrics and logs. Follow the instructions below to see how to send these different types via the three exporters.
```

Comment:
application insights -> capitalize.
BTW, I think it is time to start using "Azure Monitor" everywhere?

Reply:
Customers are used to seeing Application Insights, since all the other SDKs use the wording (e.g. Getting started with Application Insights in <language>). However, we use "Azure Monitor" everywhere in OpenCensus so this might be confusing as well. Thoughts @DawgFan ?